### PR TITLE
Encode uri for deleting keys

### PR DESCRIPTION
**Pull request by Vandesm14**
Database keys are converted to URI components before storing. When we delete a key, we aren't first encoding it to a URI component which is why it's impossible to empty a database if a key has invalid chars, such as a URL.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204051207505865

### CODE REVIEW

**Comment by nathan-pham** [comment]
Ah no wonder. Encountered this issue a while ago I figured I was just tripping over myself

File: .replit [hunk 1]
```diff
@@ -1,2 +1,79 @@
 language = "nodejs"
```
Comment (masad-frost) [inline .replit:1]: ty

File: index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Client {
    * @param {boolean} [options.raw=false] Makes it so that we return the raw string value. Default is false.
    */
   async get(key, options) {
-    return await fetch(this.key + "/" + key)
+    return await fetch(this.key + '/' + key)
```
Comment (masad-frost) [inline index.js:20]: we need to give this the same treatment, no?

**Review by masad-frost — APPROVED** [review]
thank you, can we give the same treatment to `get` with a test por favor.

**Comment by masad-frost** [comment]
We're gonna have to do a major version bump and make updates opt-in because people are probably depending on this behavior per Hyrum's law.

image

File: index.js (hunk 2 above)
Comment (Vandesm14) [reply index.js:20]: Good catch, will fix

**Comment by Vandesm14** [comment]
Added encoding/decoding for `get` and included a test for this. Lmk if there's anything else I can revise before shipping!

**Comment by masad-frost** [comment]
Ship it

**Comment by masad-frost** [comment]
Resolved in #52

**Pull request closed without merging**